test/extended/networking/network_segmentation.go

```diff
@@ -231,6 +232,19 @@ var _ = Describe("[sig-network][OCPFeatureGate:NetworkSegmentation][Feature:User
 
 						udnPodConfig.namespace = f.Namespace.Name
 
+						// make sure we don't have cgroupv1 nodes in the cluster
+						events, err := cs.CoreV1().Events("default").List(context.Background(), metav1.ListOptions{})
+						Expect(err).NotTo(HaveOccurred())
+						cgroupv1Nodes := map[string]bool{}
+						for _, event := range events.Items {
+							if event.Reason == "UDNKubeletProbesNotSupported" {
```

Comment:
the logic is relying on events of the nodes? so I assume in a cluster always there will be at least 1 event emitted at startup of that node if UDNIsolation feature gate is ON? such that by the time the test I executed we would have an event? i think there is a time limit on how many events are stored right? IIRC 1hr in default namespace? will that be enough for a 3hr test? OR is this event emitted every time someone tries to create UDNs so there will be this event happening continuously?

Reply:
Events are only generated on ovnkube startup, so there should be enough time before the test begins. As for the event lifetime, it seems to be set to 3h https://access.redhat.com/solutions/6968044 so borderline enough for the test